### ISSUE TITLE
pact-plugin-cli 0.1.1 missing linux x86_64 artifact

### ISSUE THREAD

**Issue by YOU54F** [pact-foundation/pact-plugins#52]
https://github.com/pact-foundation/pact-plugins/releases/tag/pact-plugin-cli-v0.1.1

Most peculiar - https://github.com/pact-foundation/pact-plugins/blob/4dafd503b9fe7bd980a1db923760d3c89deac977/cli/release.sh#L17

```
--- 🐿  Installing plugins CLI version '0.1.1' (from tag )
        Downloading from: https://github.com/pact-foundation/pact-plugins/releases/download/pact-plugin-cli-v0.1.1/pact-plugin-cli-linux-x86_64.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     87      0 --:--:-- --:--:-- --:--:--    88
        Downloaded /home/runner/.pact/bin/pact-plugin-cli-linux-x86_64.gz: ASCII text, with no line terminators

gzip: /home/runner/.pact/bin/pact-plugin-cli-linux-x86_64.gz: not in gzip format
make: *** [Makefile:[44](https://github.com/pact-foundation/pact-go/actions/runs/7576611778/job/20635808082#step:5:45): download_plugins] Error 1
```

**Comment by YOU54F** [comment]
So the docker build job failed but the script didn't exit as there isn't a `set -e` option in the script

https://github.com/pact-foundation/pact-plugins/actions/runs/7204827537/job/19626870293#step:8:36

which meant it didn't actually build, but the release proceeded anyway

**Comment by YOU54F** [comment]
Fixed in #58 and now available in https://github.com/pact-foundation/pact-plugins/releases/tag/pact-plugin-cli-v0.1.2
